### ISSUE TITLE
can not pruning following network with two continuous convolution block

### ISSUE THREAD

**Issue by anxu829** [Sanster/pytorch-network-slimming#5]
`class StrangeNet(nn.Module):
    def __init__(self):
        super().__init__()
        self.conv1 = nn.Conv2d(3,256,kernel_size=3)
        self.bn = nn.BatchNorm2d(256,256)
        self.conv2_1 = nn.Conv2d(256,128  , kernel_size=3)
        self.conv2_2 = nn.Conv2d(128,128 , kernel_size=3 )
        self.bn2 = nn.BatchNorm2d(128 )

    def forward(self, x):
        x = self.conv1(x)
        x = self.bn(x)
        x = self.conv2_1(x)
        x = self.conv2_2(x)
        x = self.bn2(x)
        return x`
